sdcm/cluster_k8s/__init__.py

```diff
@@ -2742,6 +2745,9 @@ def add_nodes(self,
         else:
             count = [count]
         assert len(count) in (1, len(dc_idx))
+
+        assert instance_type is None, "eks can't provision different instance types"
```

Comment:
This is common code, not EKS-specific. So, message is wrong.

Reply:
Which is fine, we don't support this for and k8s setup

Reply:
> Which is fine, we don't support this for and k8s setup

I am not sure I understand your comment.
So, please, just remove word 'eks' from the assertion message.